doc/source/user_guide/index.rst

```diff
@@ -27,9 +27,10 @@ Python code to control and monitor Ansys Fluent.
 
 Overview
 --------
-The function :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
-launches an instance of Fluent, running it as a server in the background. You can launch 
-Fluent in solution mode with no arguments:
+You use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>` method
+to launch an instance of Fluent that runs as a server in the background.
+
+You can launch Fluent in solution mode with no arguments:
```

Comment:
Is this sentence in line 9 still accurate: Setting the ``meshing_mode`` argument to ``False`` launches Fluent in solution mode.  I see "mode" being set to either "solver" or "meshing" but no Boolean "mesing_mode" argument. Also, here, we talk about meshing mode and solution mode. However, it seems to me that we should be using meshing mode and solver mode. Please advise.

Reply:
- the `meshing_mode` argument is deprecated. `launch_fluent` does accept such an argument of Boolean type
- `mode` is the new argument that will completely supersede it
- either can be provided in a `launch_fluent` call, but if both are provided an exception is raised
- "solution mode" is the same vocabulary that's used in the standard Fluent launcher